### ISSUE TITLE
Navigation Right bar button items not hiding

### ISSUE THREAD

**Issue by Gouthamansriniv** [aslanyanhaik/youtube-iOS#19]
Is there any way to remove navigation bar button items on push to next View controller i.e (table view segue navigation) and add different set of navigation bar button items with different actions.

**Comment by Abemarkar23** [comment]
Yes,  before pushing to a new View Controller what you need to do is `newVC.navigationItem.rightBarButtonItem = UIBarButtonItem(title: "title", style:  UIBarButtonItemStyle.Plain ,  target: self, action: "<action>")`

**Comment by CCLDArjun** [comment]
try
 1. `nextVC.naivgationItem.rightBarButtonItem.enabled = false`
1. `nextVC.navigationItem.rightBarButtonItem.tintColor = UIColor.clearColor()`
make sure to replace rightBarButtonItem with any one you want
